web/src/components/shared/AirgapRegistrySettings.jsx

```diff
@@ -145,10 +148,17 @@ class AirgapRegistrySettings extends Component {
   handleFormChange = (field, val) => {
     let nextState = {};
     nextState[field] = val;
+
+    if (this.props.app?.isAirgap && field === "isReadOnly" && val === false) {
```

Comment:
js could be tricky that way, how about this?
```suggestion
    if (this.props.app?.isAirgap && field === "isReadOnly" && !val) {
```